test/preload/expected/library_custom_uglify_params/my/ui/lib/library-preload.json

```diff
@@ -0,0 +1,11 @@
+{
+	"version": "2.0",
+	"name": "my.ui.lib.library-preload",
+	"modules": {
+		"my/ui/lib/library.js": "jQuery.sap.require(\"sap.ui.core.library\"),jQuery.sap.declare(\"my.ui.lib.library\"),sap.ui.getCore().initLibrary({name:\"my.ui.lib\",version:\"0.0.0\",dependencies:[\"sap.ui.core\"]});",
+		"my/ui/lib/myJS.js": "\"use strict\";function myFunction(longVariableName,longerVariableName){var sample=\"\\u518d\\u6309\\u4e00\\u6b21\\u9000\\u51fa\\u4f19\\u62fc\";return longVariableName+longerVariableName+sample}console.log(\"myJS\");",
```

Comment:
It does not seem that the copyright comments are preserved, although the default option was not changed in the target configuration (see comment in `Gruntfile.js`).